nemoguardrails/library/autoguard/README.md

```diff
@@ -0,0 +1,165 @@
+# AutoGuard
+
+This package implements the AutoGuard API integration.
+
+AutoGuard comes with a library of built-in guardrails that you can easily use:
+
```

Comment:
@abhijitpal1247 are we missing the tonal guardrail integration? if so, please include that in the integration and in the readme.

Reply:
Regarding that Rahm, what should be the output of tonal detection guardrail, should it block it, if it's negative or append the tone with the response

Reply:
resolved in prev commits